packages/terra-application-navigation/src/tabs/_TabRollup.jsx

```diff
@@ -89,7 +89,6 @@ const TabRollup = ({
       onMouseDown={disableFocusStyles}
       ref={tabRef}
       aria-current={isSelected}
-      aria-haspopup
```

Comment:
Would it make more sense to remove the role of link instead? This is the dropdown button that doesn't do any navigation when pressed.

Reply:
Since a role is required , I am guessing `button` should  work instead of link, also it is compatible with aria-haspopup

Reply:
'button' should be fine here. Though long term I think it might make more sense to use 'menubutton' combined with 'menu/menuitem' in the popup, but that could end up very different as UX is debating the viability and behavior of rollup patterns across cerner.